### PR TITLE
[OpenMP] Allow to specify what plugins to look for

### PR DESCRIPTION
**Pull request by jdoerfert**
By default we now only look for the plugins we build, but the user can overwrite that with `LIBOMPTARGET_PLUGINS_TO_LOAD="cuda,amdgpu,x86_64"`

### CODE REVIEW

**Comment by shiltian** [comment]
Can we just use arch name, such as `cuda`, `ppc64`, instead of the full name?

**Comment by jdoerfert** [comment]
> Can we just use arch name, such as `cuda`, `ppc64`, instead of the full name?

Done.

File: openmp/libomptarget/src/PluginManager.cpp [hunk 1]
```diff
@@ -23,11 +23,15 @@ PluginManager *PM;
 
 // List of all plugins that can support offloading.
 static const char *RTLNames[] = {
+#ifdef ENABLED_OFFLOAD_PLUGINS
```
Comment (shiltian) [inline openmp/libomptarget/src/PluginManager.cpp:26]: `ENABLED_OFFLOAD_PLUGINS` now is already defined right?

Comment (jdoerfert) [reply openmp/libomptarget/src/PluginManager.cpp:26]: Right. I can get rid of the alternative here.

**Review by shiltian — APPROVED** [review]
LG

**Pull request merged**